src/WorkBC.Data/SsotContext.cs

```diff
@@ -0,0 +1,208 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using WorkBC.Data.Model.Enterprise;
+
+namespace WorkBC.Data
+{
+    public partial class SsotContext : DbContext
+    {
+        private string ssotConnectionString;
+        
+        public SsotContext(DbContextOptions<SsotContext> options, IConfiguration configuration) 
+            : base(options)
+        {
+            ssotConnectionString = configuration.GetConnectionString("SsotConnection");
+        }
```

Comment:
I've created a SsotContext as switching databases is more complicated than changing the connection string for the 'EnterpriseContext'.